### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by jklaise**

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **gsunner**

If they are not already assigned, you can assign the PR to them by writing `/assign @gsunner` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: notebooks/explainer_examples.ipynb [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
-    "# Example Model Servers with Seldon"
+    "# Example Model Explanations with Seldon"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Prerequistes\n",
+    "## Prerequisites\n",
```
Comment (MarcoGorelli) [inline notebooks/explainer_examples.ipynb]: This 'Prerequistes' typo also appears in other notebooks from the `examples` folder (such as `examples/ambassador/canary/ambassador_canary.ipynb`) as well as in others from the `notebooks` folder (such as `notebooks/istio_example`)

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/896#issuecomment-536924447" title="Approved">cliveseldon</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [cliveseldon]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: notebooks/explainer_examples.ipynb (hunk 1 above)
Comment (jklaise) [reply notebooks/explainer_examples.ipynb]: We should `sed` the typo across all files.

**Pull request merged**